enhancements/machine-config/on-cluster-layering.md

```diff
@@ -0,0 +1,1137 @@
+---
+title: on-cluster-layering
+authors:
+  - cheesesashimi
+  - umohnani8
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
```

Comment:
I would expect to see some more of the MCO leadership here, as well as others who are affected by this, perhaps some input from hypershift for example? Or registry folks?